frontend/app/tab/tabbar.tsx

```diff
@@ -174,10 +174,27 @@ const TabBar = memo(({ workspace }: TabBarProps) => {
     const isFullScreen = useAtomValue(atoms.isFullScreen);
 
     const settings = useAtomValue(atoms.settingsAtom);
+    const autoHideTabsBar = settings?.["window:autohidetabsbar"] ?? false;
 
     let prevDelta: number;
     let prevDragDirection: string;
 
+    const handleAutoHideTabsBar = (event: MouseEvent) => {
```

Comment:
I'm not a big fan of onmousemove, it is very chatty and can cause the UI to use a lot of resources. I would recommend adding a full-width div fixed to the top, which onmouseenter, causes the tabbar to show. Then, you can add a onmouseleave handler to the tabbar itself to hide it again. This should remove the need for an onmousemove handler.